Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
```

Comment:
Let's just leave it as-is for now since the JDK isn't required. We can revisit in the future if necessary.